src/components/Pages/Department.js

```diff
@@ -79,6 +80,7 @@ const Department = ({
       </Head>
       {image && (
         <PageBanner
+          extraClasses="coa-PageBannerCoverDepartment"
```

Comment:
Here we pass the class `coa-PageBannerCoverDepartment ` to the PageBanner component in the Department page, so the style is specific to the department page.